### PR TITLE
Added comments to this dictionary file for GCI

### PR DESCRIPTION
**Pull request by ManitK**
Added comments to monolingual dictionary file (.lexc/.dix, .dix) in the symbol definitions area Also directed the comment reader to the Apertium wiki page on symbol definitions for more information and the Apertium wiki page of Crimean Tatar

### CODE REVIEW

File: apertium-crh.crh.lexc [hunk 1]
```diff
@@ -8,65 +9,82 @@
...
-%<cnjadv%>        ! 
+%<n%>             ! Noun                        
+%<np%>            ! Proper noun                 
+%<adj%>           ! Adjective                   
+%<prn%>           ! Pronoun                     
+%<post%>          ! Postposition                
+%<adv%>           ! Adverb                      
+%<v%>             ! Verb                        
+%<cnjcoo%>        ! Coordinating conjunction    
+%<cnjsub%>        ! Sub-ordinating conjunction
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: Fix, and restore additional comment

File: apertium-crh.crh.lexc [hunk 2]
```diff
@@ -8,65 +9,82 @@
...
+%<cnjadv%>        ! Adverbial conjunction 
 %<qst%>           ! Question word
-%<sym%>
+%<sym%>           ! Symbol
 %<cop%>           ! Copula
-%<postadv%>       ! 
-%<det%>           ! 
-%<num%>           ! 
+%<postadv%>       ! Post Adverb
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: punctuate correctly

File: apertium-crh.crh.lexc [hunk 3]
```diff
@@ -8,65 +9,82 @@
...
-%<ifi%>
-
-%<neg%>
-
-%<pass%>
-%<coop%>
-%<caus%>
-%<abil%>
+%<emph%>          ! Emphatic [imperative]
+%<ij%>		  ! Interjection
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: Get the spacing consistent.

File: apertium-crh.crh.lexc [hunk 4]
```diff
@@ -8,65 +9,82 @@
...
+
+%<nom%>           ! Nominative
+%<gen%>           ! Genitive
+%<dat%>           ! Dative
+%<acc%>           ! Accusative
+%<abl%>           ! Ablative
+%<loc%>           ! Locative
+%<ins%>           ! Instrumental=comitative 
+
 %<evid%>
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: Add comment

File: apertium-crh.crh.lexc [hunk 5]
```diff
@@ -8,65 +9,82 @@
...
-
-%<iv%>            !
-%<tv%>            !
-
-%<nec%>
-%<imp%>
-%<aor%>
-%<gno%>
-%<fut%>
-%<fut2%>          ! -EcAG
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc:50]: Why were these removed?

File: apertium-crh.crh.lexc [hunk 6]
```diff
@@ -8,65 +9,82 @@
...
-%<emph%> 
-%<ij%>
-%<nom%>           ! Nominative                  !
-%<gen%>           !
-%<dat%>           !
-%<acc%>           !
-%<loc%>           ! Locative                    ! 
-%<abl%>           ! Ablative                    !
-%<ins%>           !
-%<sim%>
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc:40]: Why was this removed?

File: apertium-crh.crh.lexc [hunk 7]
```diff
@@ -152,7 +180,6 @@ Multichar_Symbols
 %{b%}
 %{p%}
 
-%{☭%}
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc:155]: Why was this removed?

**Review by jonorthwash — CHANGES_REQUESTED** [review]

File: apertium-crh.crh.lexc (hunk 6 above)
Comment (ManitK) [reply apertium-crh.crh.lexc:40]: This isnt removed they are available under cases heading

File: apertium-crh.crh.lexc (hunk 7 above)
Comment (ManitK) [reply apertium-crh.crh.lexc:155]: Sorry added it back must have been deleted while editing.

File: apertium-crh.crh.lexc (hunk 5 above)
Comment (ManitK) [reply apertium-crh.crh.lexc:50]: They arent removed they all are present in .lexc file but they are scattered unlike above but they are present in it pls see

File: apertium-crh.crh.lexc (hunk 4 above)
Comment (ManitK) [reply apertium-crh.crh.lexc]: I am sorry what is the problem here?

File: apertium-crh.crh.lexc (hunk 3 above)
Comment (ManitK) [reply apertium-crh.crh.lexc]: Yes I will make the spacing consistent

File: apertium-crh.crh.lexc (hunk 4 above)
Comment (jonorthwash) [reply apertium-crh.crh.lexc]: This tag has no comment.  You should add one.

File: apertium-crh.crh.lexc (hunk 6 above)
Comment (jonorthwash) [reply apertium-crh.crh.lexc:40]: What line number?

File: apertium-crh.crh.lexc (hunk 5 above)
Comment (jonorthwash) [reply apertium-crh.crh.lexc:50]: What line number?

File: apertium-crh.crh.lexc (hunk 6 above)
Comment (ManitK) [reply apertium-crh.crh.lexc:40]: loc-60 
abl-59
ins-61

Comment (ManitK) [reply apertium-crh.crh.lexc:40]: sim-78

File: apertium-crh.crh.lexc (hunk 5 above)
Comment (ManitK) [reply apertium-crh.crh.lexc:50]: aor-65
gno-76
fut-77
fut2-78

File: apertium-crh.crh.lexc (hunk 2 above)
Comment (ManitK) [reply apertium-crh.crh.lexc]: it should after adverb?

File: apertium-crh.crh.lexc (hunk 4 above)
Comment (ManitK) [reply apertium-crh.crh.lexc]: Yeah added

File: apertium-crh.crh.lexc [hunk 8]
```diff
@@ -8,65 +9,85 @@
...
-
-%<neg%>
-
-%<pass%>
-%<coop%>
-%<caus%>
-%<abil%>
-%<evid%>
+%<fut2%>
+%<sim%>
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: Is this a verb tense?

File: apertium-crh.crh.lexc (hunk 4 above)
Comment (ManitK) [reply apertium-crh.crh.lexc]: Deleted that tag

File: apertium-crh.crh.lexc (hunk 8 above)
Comment (ManitK) [reply apertium-crh.crh.lexc]: I dont know abt these tags so i left them

File: apertium-crh.crh.lexc (hunk 1 above)
Comment (ManitK) [reply apertium-crh.crh.lexc]: What is the problem you are referring to?

Comment (ManitK) [reply apertium-crh.crh.lexc]: Yeah done

File: apertium-crh.crh.lexc (hunk 6 above)
Comment (jonorthwash) [reply apertium-crh.crh.lexc:40]: "sim" isn't "simile"

File: apertium-crh.crh.lexc [hunk 9]
```diff
@@ -8,159 +9,176 @@
...
-%<gen%>           !
-%<dat%>           !
-%<acc%>           !
-%<loc%>           ! Locative                    ! 
-%<abl%>           ! Ablative                    !
-%<ins%>           !
-%<sim%>
-
-%<iv%>            !
-%<tv%>            !
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc:43]: These need comments.

File: apertium-crh.crh.lexc [hunk 10]
```diff
@@ -8,159 +9,176 @@
...
-
-%<nec%>
-%<imp%>
-%<aor%>
-%<gno%>
-%<fut%>
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc:49]: This needs a comment.

File: apertium-crh.crh.lexc [hunk 11]
```diff
@@ -8,159 +9,176 @@
...
-%<qst%>           ! Question word
-%<sym%>
-%<cop%>           ! Copula
-%<postadv%>       ! 
-%<det%>           ! 
-%<num%>           ! 
-%<vaux%>          ! Auxiliary verb
-%<abbr%>          ! Abbreviation
-%<emph%> 
-%<ij%>
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc:32]: Fix the spacing on this.

File: apertium-crh.crh.lexc [hunk 12]
```diff
@@ -8,159 +9,176 @@
...
 Multichar_Symbols 
 
-%<n%>             ! Noun                        !
-%<np%>            ! Proper noun                 !
-%<adj%>           ! Adjective                   !
-%<prn%>           ! Pronoun                     !
-%<post%>          ! Postposition                !
-%<adv%>           ! Adverb                      !
-%<v%>             ! Verb                        !
-%<cnjcoo%>        ! Coordinating conjunction    !
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc:20]: Fix the extra spacing at the end of lines.

File: apertium-crh.crh.lexc (hunk 3 above)
Comment (jonorthwash) [reply apertium-crh.crh.lexc]: Please fix the spacing.

File: apertium-crh.crh.lexc [hunk 13]
```diff
@@ -8,159 +9,176 @@
...
-%<abil%>
+%<n%>             ! Noun                        	
+%<np%>            ! Proper noun                 	
+%<adj%>           ! Adjective                   	
+%<prn%>           ! Pronoun                     	
+%<post%>          ! Postposition                	
+%<adv%>           ! Adverb                      	
+%<v%>             ! Verb                        	
+%<cnjcoo%>        ! Coordinating conjunction    	
+%<cnjsub%>        ! "subordinating conjunction" (ki)
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: Why the quotation marks?

File: apertium-crh.crh.lexc [hunk 14]
```diff
@@ -8,159 +9,176 @@
...
+%<gen%>           ! Genitive	
+%<dat%>           ! Dative	
+%<acc%>           ! Accusative
+%<loc%>           ! Locative
+%<abl%>           ! Ablative		
+%<ins%>           ! Instrumental=comitative 
+%<sim%>           ! simile
+
+%<iv%>            
+%<tv%>
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: These need comments.

File: apertium-crh.crh.lexc [hunk 15]
```diff
@@ -8,159 +9,176 @@
...
+%<sim%>           ! simile
+
+%<iv%>            
+%<tv%>            
+	
+%<nec%>           ! Necessitive	
+%<imp%>           ! Imperative
+%<aor%>           ! Aorist
+%<gno%>	
+%<fut%>
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: Comments.

File: apertium-crh.crh.lexc [hunk 16]
```diff
@@ -8,159 +9,176 @@
...
 
-%<ger%>
-%<ger_pres%>
-%<ger_fut2%>
-%<ger_past%>
+%<ger%>	
+%<ger_pres%>	
+%<ger_fut2%>	
+%<ger_past%>	
 %<ger_abst%>      ! -Uv
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc:76]: These all need comments.

File: apertium-crh.crh.lexc [hunk 17]
```diff
@@ -8,159 +9,176 @@
...
 %<gpr_pot%>
 
-%<gna_while%>
-%<gna_perf%>
-%<gna_when%>
-%<gna_neg%>
+%<gna_while%>	
+%<gna_perf%>	
+%<gna_when%>	
+%<gna_neg%>
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: These all need comments.  Also, why did you add spacing to the end of the lines??

File: apertium-crh.crh.lexc [hunk 18]
```diff
@@ -8,159 +9,176 @@
...
- 
-%>                !
-%+
+%<px1sg%>         ! First person singular	
+%<px2sg%>         ! Second person singular	
+%<px3sg%>         ! Third person singular	
+%<px3sp%>         ! Third person singular/plural	
+%<px3pl%>         ! Third person plural	
+%<px1pl%>         ! First person plural	
+%<px2pl%>         ! Second person plural
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc:104]: These are possessives.

File: apertium-crh.crh.lexc [hunk 19]
```diff
@@ -8,159 +9,176 @@
...
+%<attr%>          ! Attributive	
+%<subst%>         ! Substantive	
+%<advl%>          ! Adverbial	
+
+%<pers%>          ! Personal	
+%<dem%>           ! Demonstrative	
+%<ind%>           ! Indefinite	
+%<itg%>           ! Interrogative	
+%<qnt%>           ! Quantifier
+%<ord%>           ! Ordinal
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: Maybe add a comment above this block saying what these are for.

File: apertium-crh.crh.lexc [hunk 20]
```diff
@@ -8,159 +9,176 @@
...
+
+%<cog%>           ! Cognomen (family name)	
+%<ant%>           ! Anthroponym (person name)	
+%<pat%>           ! Patronymic	
+%<org%>           ! Organisation	
+%<al%>            ! Others
+%<top%>           ! Toponym (place name)
+
+%<percent%>       ! Percent(%)	
+%<hash%>          ! Hash(#)
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: spaces before parentheses

File: apertium-crh.crh.lexc [hunk 21]
```diff
@@ -8,159 +9,176 @@
...
+
+%<m%>             ! Masculine    	
+%<f%>             ! Feminine     	
+%<mf%>            ! Either
+
+%<cog%>           ! Cognomen (family name)	
+%<ant%>           ! Anthroponym (person name)	
+%<pat%>           ! Patronymic	
+%<org%>           ! Organisation	
+%<al%>            ! Others
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc:137]: "Others" what?

File: apertium-crh.crh.lexc [hunk 22]
```diff
@@ -8,159 +9,176 @@
...
+%>                !	
+%+		  !
+
+
+%<part%>          ! Particle 	
+%<emph%>          ! Emphatic [imperative]	
+%<cond%>          ! Conditional  ! Шарт  ! Условное  !! -с{A}		
+%<rec%>           ! Reciprocal (?)	
+%<iver%>	  ! suffix conveying ease/speed of action	
+%<eyaz%>	  ! suffix conveying the action *almost* happened
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: spacing

File: apertium-crh.crh.lexc [hunk 23]
```diff
@@ -8,159 +9,176 @@
...
-%<ger_past%>
+%<ger%>	
+%<ger_pres%>	
+%<ger_fut2%>	
+%<ger_past%>	
 %<ger_abst%>      ! -Uv
 
-%<gpr_fut%>
-%<gpr_past%>
+%<gpr_rsub%>      ! past-tense subject-relative verbal adjective
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: Where is this from?

File: apertium-crh.crh.lexc [hunk 24]
```diff
@@ -8,159 +9,176 @@
...
+%<gna_neg%>	
 
-%<prc_perf%>      ! -Ip 
-%<prc_fut%>       ! 
-%<prc_impf%>      ! 
-%<prc_cond%>      ! 
+%<prc_perf%>      ! -Ip 	
+%<prc_fut%>       ! 	
+%<prc_impf%>      ! 	
+%<prc_cond%>      !
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: What's all the extra space for???

File: apertium-crh.crh.lexc [hunk 25]
```diff
@@ -8,159 +9,176 @@
...
+%{b%}	
+%{p%}	
+%{☭%}	
+%{ъ%}             ! force back-vowel harmony	
+%{ь%}             ! force front-vowel harmony	
+%-	
+%% 	
+ 	
+%>                !	
+%+		  !
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: No comments on these two?

File: apertium-crh.crh.lexc [hunk 26]
```diff
@@ -8,159 +9,176 @@
...
+%{C%}             !	
+%{I%}             ! Can only be unrounded: i, ı	
+%{Y%}             ! Can be rounded or unrounded: i, ü, ı, u	
+%{i%}             !	
+%{l%}             !	
+%{s%}             !	
+%{y%}             !	
+%{j%}             !	
+%{n%}             !	
+%{e%}
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: Why did you remove the newlines between these?

File: apertium-crh.crh.lexc [hunk 27]
```diff
@@ -8,159 +9,176 @@
...
+%{j%}             !	
+%{n%}             !	
+%{e%}	
+%{ö%}	
+%{o%}	
+%{ı%}	
+%{b%}	
+%{p%}	
+%{☭%}	
+%{ъ%}             ! force back-vowel harmony
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: You removed a newline here too.

File: apertium-crh.crh.lexc [hunk 28]
```diff
@@ -8,159 +9,176 @@
...
+%<rec%>           ! Reciprocal (?)	
+%<iver%>	  ! suffix conveying ease/speed of action	
+%<eyaz%>	  ! suffix conveying the action *almost* happened	
+%<dub%>           ! dubative -mIş	
+%<abstr%>         ! Abstract suffix	
+%<nec%>           ! Necessitive	
+%<ref%>           ! Reflexive	
+%<recip%>         ! Reciprocal	
+%<coll%>          ! Collective	
+%<dist%>          ! Distibutive
```
Comment (jonorthwash) [inline apertium-crh.crh.lexc]: Where did you get these from?  Some of them are duplicates, and some of them are not in the file....

**Review by jonorthwash — CHANGES_REQUESTED** [review]

File: apertium-crh.crh.lexc (hunk 6 above)
Comment (ManitK) [reply apertium-crh.crh.lexc:40]: oh okay i will change that

File: apertium-crh.crh.lexc (hunk 9 above)
Comment (ManitK) [reply apertium-crh.crh.lexc:43]: ok will comment on them

File: apertium-crh.crh.lexc (hunk 10 above)
Comment (ManitK) [reply apertium-crh.crh.lexc:49]: Yeah aor and imp is already added and I have commented on fur

Comment (ManitK) [reply apertium-crh.crh.lexc:49]: *fut

File: apertium-crh.crh.lexc (hunk 12 above)
Comment (ManitK) [reply apertium-crh.crh.lexc:20]: Ok

File: apertium-crh.crh.lexc (hunk 13 above)
Comment (ManitK) [reply apertium-crh.crh.lexc]: Ok I will remove them

File: apertium-crh.crh.lexc (hunk 17 above)
Comment (ManitK) [reply apertium-crh.crh.lexc]: Ok will add comments

File: apertium-crh.crh.lexc (hunk 24 above)
Comment (ManitK) [reply apertium-crh.crh.lexc]: Nothing in particular

File: apertium-crh.crh.lexc (hunk 19 above)
Comment (ManitK) [reply apertium-crh.crh.lexc]: Okay sure

File: apertium-crh.crh.lexc (hunk 27 above)
Comment (ManitK) [reply apertium-crh.crh.lexc]: ok sorry

File: apertium-crh.crh.lexc (hunk 28 above)
Comment (ManitK) [reply apertium-crh.crh.lexc]: Resolved this issue

File: apertium-crh.crh.lexc (hunk 23 above)
Comment (ManitK) [reply apertium-crh.crh.lexc]: Why is this wrong?

File: apertium-crh.crh.lexc (hunk 15 above)
Comment (ManitK) [reply apertium-crh.crh.lexc]: I dont know what is gno i have commented on fut

**Comment by jonorthwash** [comment]
The main remaining issues:
- There are extra spaces at the end of a lot of lines.
- Spacing is inconsistent in several places, some of which I've pointed out multiple times.
- There are a few tags that seem to have been removed.
- There are a few tags that seem to have been added.
- A number of labels are missing still.  You haven't asked about what any of these should be.
- Some tags are incorrect, especially `<emph>` and `<qst>`.  These aren't for words, but clitics.
- The non-finite verb categories should be labelled better, at least by section (e.g., `gna` types should have a heading saying what they are)

**Comment by ManitK** [comment]
- There are no new tags added.

- Extra spacing has been dealt with.

- The spacing is consistent everywhere I guess please see where it isn't according to you.

- I have added headings wherever I was able to.

**Comment by ManitK** [comment]
The tags which I do not know about are:

- `gno`	

- `ger_pres`

- `gna_while`

- `C`

- `j`

**Comment by jonorthwash** [comment]
>     * There are no new tags added.

There had been.  You seem to have fixed that.

>     * Extra spacing has been dealt with.

Nope, there are still problems.

>     * The spacing is consistent everywhere I guess please see where it isn't according to you.

Yes, I see those places.  Could you fix them?

>     * I have added headings wherever I was able to.

Good, thanks.

**Comment by jonorthwash** [comment]
> The tags which I do not know about are:
> 
>     * `gno`

Gnomic tense/aspect

>     * `ger_pres`

It's a combination between `<ger>` and `<pres>`, both of which are defined.  But yes, it appaers not to be used.  You can just comment it out.

>     * `gna_while`

Comment this out too.

> 
>     * `C`
> 
>     * `j`

Do you mean `{C}` and `{j}`

Don't change the archiphoneme definitions.  You've added a bunch of definitions that are incorrect for this language.  Your task is to worry about the morphological tags, not the archiphonemes.

Also, the forms you specified for many of the non-finite tags are wrong / seem to be from other languages.  Please make them consistent with Crimean Tatar.

**Comment by ManitK** [comment]
I had left the archiphoneme out but when you said that a lot of tags are not commented on so I explained them also.

**Comment by ManitK** [comment]
Which non-finite tags are those can you please tell?

**Comment by ManitK** [comment]
Is it ok if I leave out `gpr_rsub` because it has not been used?

**Comment by ManitK** [comment]
Also please help me with this, so can you compare these lines:
38,56,68,71 and 180
First see those lines here:https://github.com/ManitK/apertium-crh/edit/master/apertium-crh.crh.lexc
and then compare with those lines here:https://github.com/apertium/apertium-crh/pull/2/files
I think these are the only ones whose spacing is inconsistent.

File: apertium-crh.crh.lexc [hunk 29]
```diff
@@ -18,107 +23,119 @@ Multichar_Symbols
...
+%<qst%>           ! Modal question particle     !# м{I}
+                  
+%<sym%>           ! Symbol
 %<cop%>           ! Copula
-%<postadv%>       ! 
-%<det%>           ! 
-%<num%>           ! 
+%<postadv%>       ! Post of Adverb
+%<det%>           ! Determiner
+%<num%>           ! Numeral
```
Comment (ManitK) [inline apertium-crh.crh.lexc]: Why are these words highlighted?

Comment (ManitK) [inline apertium-crh.crh.lexc:34]: Why are these words highlighted?

File: apertium-crh.crh.lexc [hunk 30]
```diff
@@ -18,107 +23,119 @@ Multichar_Symbols
...
-%<pass%>
-%<coop%>
-%<caus%>
-%<abil%>
-%<evid%>
+%<pass%>          ! Passive
+%<coop%>          ! Cooperative
+%<caus%>          ! Causative
+%<abil%>          ! Abilative
+%<evid%>          ! Evidential
```
Comment (ManitK) [inline apertium-crh.crh.lexc:69]: Why are these words highlighted?

File: apertium-crh.crh.lexc [hunk 31]
```diff
@@ -18,107 +23,119 @@ Multichar_Symbols
...
 %<ger_abst%>      ! -Uv
 
-%<gpr_fut%>
-%<gpr_past%>
-%<gpr_fut2%>
-%<gpr_pot%>
+! Verbal adjectives
+%<gpr_rsub%>      !
+%<gpr_fut%>       ! -{I}r/-{A}r
+%<gpr_past%>      ! -{G}{A}n
```
Comment (ManitK) [inline apertium-crh.crh.lexc:81]: Why are these words highlighted?Same goes for lines 132,135,136 and 148

File: apertium-crh.crh.lexc (hunk 29 above)
Comment (mr-martian) [reply apertium-crh.crh.lexc:34]: the highlighting is due to `git diff` recognizing that only part of a line has been changed

Comment (ManitK) [reply apertium-crh.crh.lexc:34]: Can I remove this highlighting and if yes how?

Comment (mr-martian) [reply apertium-crh.crh.lexc:34]: That highlighting is solely a matter of the interface on which you are viewing the changes. I am not aware of any way to turn it off.

Comment (ManitK) [reply apertium-crh.crh.lexc:34]: Ok thanks anyways

File: apertium-crh.crh.lexc [hunk 32]
```diff
@@ -18,107 +23,119 @@ Multichar_Symbols
...
+%<tv%>            ! Intransitive
+
+%<nec%>           ! Necessitive
+%<imp%>           ! Imperative
+%<aor%>           ! Aorist
+%<gno%>           ! Gnomic tense
+%<fut%>           ! -{I}р
 %<fut2%>          ! -EcAG
-%<past%>
-%<ifi%>
```
Comment (mr-martian) [inline apertium-crh.crh.lexc:52]: These two tags have been removed and should be added back.

File: apertium-crh.crh.lexc [hunk 33]
```diff
@@ -18,107 +23,119 @@ Multichar_Symbols
...
-%<sg%>            !
-%<pl%>            ! Plural                      !
-
-%<attr%>          ! Attributive                 !
-%<subst%>         ! Substantive                 !
-%<advl%>          ! 
-
-%<pers%>          ! Personal                    !
-%<dem%>
-%<ref%>
```
Comment (mr-martian) [inline apertium-crh.crh.lexc:103]: This tag was removed and should be added back.

**Review by mr-martian — CHANGES_REQUESTED** [review]

File: apertium-crh.crh.lexc (hunk 32 above)
Comment (ManitK) [reply apertium-crh.crh.lexc:52]: Ok added them back and commented on them.

File: apertium-crh.crh.lexc (hunk 33 above)
Comment (ManitK) [reply apertium-crh.crh.lexc:103]: Sure added back
